src/components/network-modification-tree-pane.jsx

```diff
@@ -39,7 +39,7 @@ import {
     fetchNetworkModificationTreeNode,
     fetchStashedNodes,
 } from '../services/study/tree-subtree';
-import { buildNode, getUniqueNodeName, unbuildNode } from '../services/study';
+import { buildNode, getUniqueNodeName, unbuildNode } from '../services/study/index';
```

Comment:
```suggestion
import { buildNode, getUniqueNodeName, unbuildNode } from '../services/study';
```
"index" is always implicit in imports.

Reply:
Done